components/ShoppingList/ShoppingList.test.js

```diff
@@ -5,27 +5,24 @@ import ShoppingList from "./ShoppingList";
 const shoppingListItems = [
   {
     id: "a",
-    item_id: "1",
-    name: "Brot",
+    item: { id: "1", name: "Brot" },
     checked: false,
   },
   {
     id: "b",
-    item_id: "2",
-    name: "Butter",
-    checked: true,
+    item: { id: "1", name: "Bananen" },
+    checked: false,
   },
   {
     id: "c",
-    item_id: "3",
-    name: "Nudeln",
-    checked: false,
+    item: { id: "1", name: "Milch" },
+    checked: true,
   },
 ];
 
 describe("ShoppingList", () => {
   it("renders all items from current shopping list", () => {
-    render(<ShoppingList items={shoppingListItems} />);
+    render(<ShoppingList listItems={shoppingListItems} />);
 
     const shoppingItems = screen.getAllByRole("listitem");
     const itemBrot = screen.getByText(/Brot/i);
```

Comment:
Hey Angie, I am not really good with regular expressions, but is this one needed in here? I think you could also just write "Brot"

Reply:
Hey,
good point! Also, I DONT want it to be case insensitive at this point, because my shopping list shouldn't display `brot` or `bRot` etc.
TY for the tipp!! :)